core/src/main/scala/org/apache/spark/ui/ToolTips.scala

```diff
@@ -99,4 +99,16 @@ private[spark] object ToolTips {
        dynamic allocation is enabled. The number of granted executors may exceed the limit
        ephemerally when executors are being killed.
     """
+
+  val THRIFT_SERVER_FINISH_TIME =
```

Comment:
Could we move these changes and tests to `hive-thriftserver` module? @srowen @dongjoon-hyun What do you think?

Reply:
Yeah, if these are only relevant to the thrift server UI, they could go with the other thrift server UI code.

Reply:
+1, too.

Reply:
Ok, it make sense to have the tooltips in thriftserver module, i'm going to move it.
To generate the tables for this pages we use UIUtils from spark core module so i need to improve this utils to add this new functionality. 
Also, this improvements could be use in other pages to add tooltips to the tables.